src/components/SelectionList/SearchTableHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {View} from 'react-native';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import CONST from '@src/CONST';
+
+type SearchTableHeaderProps = {
+    /** Whether we should show the merchant or description column */
+    shouldShowMerchant: boolean;
+};
+
+function SearchTableHeader({shouldShowMerchant}: SearchTableHeaderProps) {
+    const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+    const {isSmallScreenWidth, isMediumScreenWidth} = useWindowDimensions();
+    const {translate} = useLocalize();
+    const displayNarrowVersion = isMediumScreenWidth || isSmallScreenWidth;
+
+    if (displayNarrowVersion) {
+        return;
+    }
```

Comment:
isSmallScreen is the typical variable name we'd use to indicate mobile size

Reply:
Well, this includes both small and medium size screens, what name do we use for that?

Reply:
Gonna use isLargeScreenWidth instead

Reply:
Done